examples/jsm/loaders/3DMLoader.js

```diff
@@ -615,7 +624,7 @@ class Rhino3dmLoader extends Loader {
 
 				} else if ( geometry.isLinearLight ) {
 
-					console.warn( 'THREE.3DMLoader:  No conversion exists for linear lights.' );
+					self.postMessage( { type: 'warning', id: taskID, message: 'THREE.3DMLoader:  No conversion exists for linear lights.' } );
```

Comment:
@karimi here we are no longer in the worker. What do you want to do with these warnings? I'm don't think posting messages is the way to go here.